files/setup.sh

```diff
@@ -23,6 +23,8 @@ ENABLE_SSH=$(/root/setup/getOvfProperty.py "guestinfo.enable_ssh" | tr '[:upper:
 VCENTER_SERVER=$(/root/setup/getOvfProperty.py "guestinfo.vcenter_server")
 VCENTER_USERNAME=$(/root/setup/getOvfProperty.py "guestinfo.vcenter_username")
 VCENTER_PASSWORD=$(/root/setup/getOvfProperty.py "guestinfo.vcenter_password")
+VCENTER_USERNAME_FOR_VEBA_UI=$(/root/setup/getOvfProperty.py "guestinfo.vcenter_veba_ui_username")
```

Comment:
Regarding my comment above: does `getOvfProperty` correctly handle special characters so we don't have to worry about it when we create the secret above?

Reply:
Good point. `getOvfProperty` only correctly handles the pass through of the value as there [was an issue](https://github.com/vmware-samples/vcenter-event-broker-appliance/pull/283) awhile back but doesn't cover the special character.